sopel_github/plugin.py

```diff
@@ -436,35 +435,23 @@ def github_repo(bot, trigger):
         return bot.reply('I need a repository name, or `user/reponame`.')
 
     if repo.lower() == 'version':
-        return bot.say('[GitHub] Version {} by {}, report issues at {}'.format(
-            github_plugin.__version__, github_plugin.__author__, github_plugin.__repo__))
+        pfx = bot.settings.core.help_prefix
+        msg = "Sorry, the `{pfx}gh version` command is deprecated. "
+
+        if bot.has_plugin('version'):
+            msg += "Try `{pfx}version github` instead!"
+        else:
+            msg += "Ask my owner about the `version` plugin!"
+
+        bot.reply(msg.format(pfx=pfx))
+        return plugin.NOLIMIT
 
     if repo.lower() == 'status':
-        current = json.loads(requests.get('https://status.github.com/api/status.json').text)
-        lastcomm = json.loads(requests.get('https://status.github.com/api/last-message.json').text)
-
-        status = current['status']
-        if status == 'major':
-            status = "\x02\x034Broken\x03\x02"
-        elif status == 'minor':
-            status = "\x02\x037Shakey\x03\x02"
-        elif status == 'good':
-            status = "\x02\x033Online\x03\x02"
-
-        lstatus = lastcomm['status']
-        if lstatus == 'major':
-            lstatus = "\x02\x034Broken\x03\x02"
-        elif lstatus == 'minor':
-            lstatus = "\x02\x037Shakey\x03\x02"
-        elif lstatus == 'good':
-            lstatus = "\x02\x033Online\x03\x02"
-
-        timezone = get_timezone(bot.db, bot.config, None, trigger.nick)
-        if not timezone:
-            timezone = 'UTC'
-        lastcomm['created_on'] = format_time(bot.db, bot.config, timezone, trigger.nick, trigger.sender, from_utc(lastcomm['created_on']))
-
-        return bot.say('[GitHub] Current Status: ' + status + ' | Last Message: ' + lstatus + ': ' + lastcomm['body'] + ' (' + lastcomm['created_on'] + ')')
+        # This was previously more complex, but broke sometime before June 2024.
+        # It could be re-improved using other API endpoints documented here:
```

Comment:
As someone who doesn't keep up with this plugin, it's a little unclear from this comment what exactly broke, but as long as the maintainer knows what this means it's okay 😜

Reply:
The maintainer (me) _doesn't_ actually know 😅 — all I know is "it used to output more information than just the current status". Since it no longer works, I can't go back and look at what the `last-message.json` endpoint used to do.

It might have been an old way of getting the latest "incident", and that's my best guess. Really, linking to the status API docs is punting a design decision down the road. Goal here was to go from "broken since ????" to "does _something_" in the shortest amount of time possible, so as not to delay a Sopel-8-compatible plugin release.

I should just write up an issue about the `status` feature summarizing the available API data, when I have a minute.